ocs_ci/deployment/vmware.py

```diff
@@ -618,6 +630,16 @@ def destroy_cluster(self, log_level="DEBUG"):
         """
         previous_dir = os.getcwd()
 
+        # Download terraform binary based on ocp version and
+        # update in ENV_DATA
+        # use "0.11.14" for releases below OCP 4.5
+        ocp_version = get_ocp_version()
+        terraform_version = constants.TERRAFORM_OLD_VERSION
```

Comment:
Write TODO in comment with something like:
```
# TODO: For cluster installed by old version of terraform we need to still run old version for destroy
```
And open issue for this please.

Reply:
Ah, I see you've already opened issue here: https://github.com/red-hat-storage/ocs-ci/issues/2399
So you can put it to the comment in TODO here